Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ListViewExtensions/ListViewExtensionsPage.xaml.cs

```diff
@@ -15,29 +14,95 @@ namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
 {
     public sealed partial class ListViewExtensionsPage : Page, IXamlRenderListener
     {
+        private ListView sampleListView;
+        private TextBlock indexInput;
+        private TextBlock itemPlacementInput;
+        private CheckBox disableAnimationInput;
+        private CheckBox scrollIfVisibileInput;
+        private TextBlock additionalHorizontalOffsetInput;
+        private TextBlock additionalVerticalOffsetInput;
+
         public ListViewExtensionsPage()
         {
             this.InitializeComponent();
+            Load();
         }
 
-        public ICommand SampleCommand => new DelegateCommand<PhotoDataItem>(OnExecuteSampleCommand);
-
-        public async void OnXamlRendered(FrameworkElement control)
+        public void OnXamlRendered(FrameworkElement control)
         {
-            var sampleListView = control.FindChild("SampleListView") as ListView;
+            sampleListView = control.FindChild("SampleListView") as ListView;
+            indexInput = control.FindChild("IndexInput") as TextBlock;
+            itemPlacementInput = control.FindChild("ItemPlacementInput") as TextBlock;
+            disableAnimationInput = control.FindChild("DisableAnimationInput") as CheckBox;
+            scrollIfVisibileInput = control.FindChild("ScrollIfVisibileInput") as CheckBox;
+            additionalHorizontalOffsetInput = control.FindChild("AdditionalHorizontalOffsetInput") as TextBlock;
+            additionalVerticalOffsetInput = control.FindChild("AdditionalVerticalOffsetInput") as TextBlock;
 
             if (sampleListView != null)
             {
-                sampleListView.ItemsSource = await new Data.PhotosDataSource().GetItemsAsync();
+                sampleListView.ItemsSource = GetOddEvenSource(201);
             }
 
             // Transfer Data Context so we can access SampleCommand
             control.DataContext = this;
         }
 
-        private async void OnExecuteSampleCommand(PhotoDataItem item)
+        private void Load()
+        {
+            SampleController.Current.RegisterNewCommand("Start Smooth Scroll", (sender, args) =>
+            {
+                var index = int.Parse(indexInput?.Text);
```

Comment:
This is crashing when viewing from the Properties panel. Looks like `indexInput.Text` is blank. Probably because it's binding an expected integer to the TextBlock Text field. It's a bit weird to have all the hidden elements in the bind file, but I understand why they were put there. We don't have the best setup for this outside of putting them on the main page of the sample outside a XAML Root, but then that doesn't feel integrated.

However, we should at least bind it to a slider itself or maybe a NumberBox `Value` instead so that it binds properly from the slider we have on the Property panel? Otherwise, I'm not sure what might be going on here.

Reply:
I updated the sample a bit, let me know what you think.